src/icons/PhoneIcon.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+import { Icon } from '../components/Icon'
+
+const PhoneIcon = (props) => (
+  <Icon { ...props }>
+    <g>
```

Comment:
we need to shift it at some point because it isn't exactly centered in the svg viewbox - I just forgot to add the translation after manipulating in chrome